### DISCUSSION TITLE
Mask cutting

### DISCUSSION THREAD

**Discussion by ProgramClass2** [TurboWarp/extensions#1188] [Ideas]
It is possible to make an extension that only shows or hides certain parts of the stage/sprite?
Kind of like Clipping & Blending but using costumes instead of blocks limited to making rectangles. I think this would open many possibilities, like making it compatible with the camera extension since it would use costumes.

**Comment by SharkPool-SP** [comment]
I know image and sprite effects can do that :D

https://sharkpool-sp.github.io/SharkPools-Extensions/

**Reply by ProgramClass2** [reply]
How have I not come across this library before??? These are **amazing** extensions! :)

**Reply by SharkPool-SP** [reply]
> How have I not come across this library before??? These are **amazing** extensions! :)

Thanks!!
